orc/orcarm.c

```diff
@@ -109,19 +122,58 @@ orc_arm_emit_push (OrcCompiler *compiler, int regs, orc_uint32 vregs)
   if (regs) {
     int x = 0;
 
-    ORC_ASM_CODE(compiler,"  push {");
-    for(i=0;i<16;i++){
-      if (regs & (1<<i)) {
-        x |= (1<<i);
-        ORC_ASM_CODE(compiler,"r%d", i);
-        if (x != regs) {
-          ORC_ASM_CODE(compiler,", ");
+    if (compiler->is_64bit) {
+      int count, stores;
+      int stack_increased = 0;
+      /** a number of 1s in regs */
+      count = count_reg_ones (regs);
+      /** AArch64 requires a 16-byte aligned stack pointer */
+      stores = (count-1)/2+1;
+      x = -1;
+      for(i=0;i<32;i++){
```

Comment:
I think ORC uses GST coding convention.

Reply:
Ok.. for relocated codes, keep their style.
gst-indent script fixes styles for GST coding style; however, orc/arm code appears to be way too much diverted. So don't fix the styles of original code.

Reply:
Oh, thanks. I will run gst-indent script.